src/components/Settings/SettingsServices.tsx

```diff
@@ -68,40 +74,48 @@ const ServerInstance: React.FC<ServerInstanceProps> = ({
             <h3 className="font-medium leading-5 text-white truncate">
               {name}
             </h3>
-            {isDefault && (
-              <Badge>
-                <FormattedMessage {...messages.default} />
-              </Badge>
-            )}
+            {isDefault && <Badge>{intl.formatMessage(messages.default)}</Badge>}
             {isDefault4K && (
               <Badge badgeType="warning">
-                <FormattedMessage {...messages.default4k} />
+                {intl.formatMessage(messages.default4k)}
               </Badge>
             )}
             {isSSL && (
               <Badge badgeType="success">
-                <FormattedMessage {...messages.ssl} />
+                {intl.formatMessage(messages.ssl)}
               </Badge>
             )}
           </div>
           <p className="mt-1 text-sm leading-5 text-gray-300 truncate">
             <span className="mr-2 font-bold">
-              <FormattedMessage {...messages.address} />
+              {intl.formatMessage(messages.address)}
             </span>
-            {address}
+            {isSSL ? 'https://' : 'http://'}
+            {address}:{port}
           </p>
           <p className="mt-1 text-sm leading-5 text-gray-300 truncate">
             <span className="mr-2 font-bold">
-              <FormattedMessage {...messages.activeProfile} />
-            </span>{' '}
+              {intl.formatMessage(messages.activeProfile)}
+            </span>
             {profileName}
           </p>
         </div>
-        <img
-          className="flex-shrink-0 w-10 h-10"
-          src={`/images/${isSonarr ? 'sonarr' : 'radarr'}_logo.svg`}
-          alt=""
-        />
+        {externalUrl && (
+          <a href={externalUrl} className="opacity-50 hover:opacity-100">
+            <img
+              className="flex-shrink-0 w-10 h-10"
+              src={`/images/${isSonarr ? 'sonarr' : 'radarr'}_logo.svg`}
+              alt=""
+            />
+          </a>
+        )}
```

Comment:
External URL isn't required for links to sonarr/radarr to work. Some people don't need them in their setup because everything is only internal anyways 🤷

Reply:
@sct Ah true.  I guess I can build the URL outside of the `return` and use the internal if external isn't set.